plugins/lookup/onepassword_raw.py

```diff
@@ -30,6 +30,11 @@
         description: Item section containing the field to retrieve (case-insensitive). If absent will return first match from any section.
       subdomain:
         description: The 1Password subdomain to authenticate against.
+      domain:
+        description: Domain of 1Password. Default is U(1password.com).
```

Comment:
It's already shown next to the option.
```suggestion
        description: Domain of 1Password.
```